vendor.conf

```diff
@@ -2,7 +2,7 @@ github.com/blang/semver v3.1.0
 github.com/boltdb/bolt e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/containerd/cgroups 9c238e632e80d94f71a067c3deb9b34b1886ef18
-github.com/containerd/containerd 12c79cc0d9a38f74420ee669e44d551fda664fd4
+github.com/containerd/containerd 04659d94051126f4fe4fcddf068550db447ff5a0
```

Comment:
we have a hack versions file as well that also has to change with this version..

Reply:
github.com/containerd/cgroups f7dd103d3e4e696aa67152f6b4ddd1779a3455a9 

see: https://github.com/containerd/containerd/blame/master/vendor.conf#L4

Reply:
github.com/opencontainers/runc 74a17296470088de3805e138d3d87c62e613dfc4

here and also in the versions file..

if we move up to the latest containerd we also move up to the latest runc and container's dependencies.. this is because we use the containerd client which gets bound to our go routines...

Reply:
So what we do here is look at containerd's blame and check for recently updated vendor files.. then vendor that over as well.

Cheers!

Reply:
amended, does a script in hack need to be called with the versions update?

Reply:
no.. the vendor script in hack/ is just for "sorting" if you added something new.

Reply:
ammended, additionally added golang.org/x/sys update.